### PR TITLE
Add support for Experimental Framework

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR add support for the Experimental Framework coming soon.
See https://github.com/jpchase/ExperimentalFramework/blob/master/explainer.md for some background.

screenshot 2016-02-03 at 10 41 01 am

### CODE REVIEW

File: static/elements/chromedash-feature.html [hunk 1]
```diff
@@ -84,6 +84,11 @@
             <iron-icon icon="icons:extension" class="experimental"></iron-icon>
           </span>
         </template>
+        <template is="dom-if" if="{{experimentalframework}}">
```
Comment (ebidel) [inline static/elements/chromedash-feature.html]: instead of a defining a new prop, you can just use `feature.meta. experimentalframework` directly.

**Comment by beaufortfrancois** [comment]
Thank you @ebidel!
I've updated my PR to not define a new prop.

File: static/elements/chromedash-featurelist.html [hunk 2]
```diff
@@ -20,6 +20,7 @@
                class="milestone-marker">{{_computeMilestoneString(feature.meta.milestone_str)}}</div>
           <li id="{{feature.id}}" is="chromedash-feature"
                feature="[[feature]]" needsflag="[[feature.meta.needsflag]]"
+               experimentalframework="[[feature.meta.experimentalframework]]"
```
Comment (ebidel) [inline static/elements/chromedash-featurelist.html]: you can get rid of this now.

**Comment by ebidel** [comment]
Reviewed

**Comment by beaufortfrancois** [comment]
Oops... thanks for spotting this.
PR updated.

**Comment by ebidel** [comment]
:+1:

**Comment by beaufortfrancois** [comment]
Please let me know when it's updated.

**Comment by ebidel** [comment]
Live!

**Comment by jyasskin** [comment]
In https://www.chromestatus.com/admin/features/edit/5264933985976320, "In experimental framework" appears below "Removed" in the "Status in Chrome:" dropdown. Is that the right place for it?

**Comment by ebidel** [comment]
Yea, @beaufortfrancois might be good to move this above Deprecated or something.

**Comment by beaufortfrancois** [comment]
Here it is: https://github.com/GoogleChrome/chromium-dashboard/pull/264

**Pull request merged**